src/index.js

```diff
@@ -90,6 +135,17 @@ const extendCodeBlockWithSyntaxHighlighting = ( settings ) => {
 						placeholder={ __( 'Write code…', 'syntax-highlighting-code-block' ) }
 						aria-label={ __( 'Code', 'syntax-highlighting-code-block' ) }
 					/>
+					<div aria-hidden={ true } className="code-block-bg">
+						{ attributes.content.split( '\n' ).map( ( v, i ) => {
```

Comment:
Since I work with PHP so much, I always forget that in JS `'\n' === "\n"`. I wonder if this would be better as:

```suggestion
						{ attributes.content.split( /\n/ ).map( ( v, i ) => {
```

Probably zero difference and purely style preference.